src/libraries/System.Private.CoreLib/src/System/Reflection/StrongNameKeyPair.cs

```diff
@@ -6,25 +6,14 @@
 
 namespace System.Reflection
 {
+    [Obsolete(Obsoletions.StrongNameKeyPairMessage, DiagnosticId = Obsoletions.StrongNameKeyPairDiagId)]
```

Comment:
I saw that but didn't put that because there is no doc corresponding to this obsoletion and doesn't seem to need to have one as  there is nothing really put into the doc. Compared to the other obsoletions it doesn't have any implementation and no work around/alternative (as far asi know). So there is nothing to put in the doc expect the message that it is not supported in .NetCore, i am planning to add a note in the existing API doc, but just didn't felt we need separate doc for this obsoletion, please let me know if we still want the doc url

Reply:
I do think we want to have docs for each obsoletion for discoverability, even if the doc is short. @GrabYourPitchforks -- do you agree?

Reply:
If not the same id for all throwing PNSE I guess we would at least group them by the relevance/type and give separate ids for each group. Then for this PR only one ID will be added for `ReflectionOnly loading`

Reply:
> Then for this PR only one ID will be added for `ReflectionOnly loading`

That sounds good and matches the pattern we've been following with the other obsoletions.

Reply:
Updated all, thanks!